torch_geometric/nn/functional/spmm.py

```diff
@@ -0,0 +1,22 @@
+import torch
+from torch import Tensor
+from torch_sparse import SparseTensor
+
+from torch_geometric.utils import is_torch_sparse_tensor
+
+
+def spmm(src: SparseTensor, other: Tensor, reduce: str = "sum") -> Tensor:
```

Comment:
Can we make this TorchScript compatible via the `overload` decorator?

Reply:
Sure :)